client.go

```diff
@@ -390,6 +392,27 @@ type Client struct {
 // Initialize a default *Client instance
 var DefaultClient = newClient(nil)
 
+func (c *Client) SetIgnoreErrors(errs []string) error {
+	joinedRegexp := strings.Join(errs, "|")
```

Comment:
I think we should wrap this in the mutex so it can safely be called concurrently if it were to be somehow. We do this with all the other config settings.